legal-api/report-templates/dissolution.html

```diff
@@ -27,7 +27,7 @@
             </tr>
          </table>
          <div class="business-name-header">
-                <label class="lbl-business-name">{{business.businessName}}</label>
+                <label class="lbl-business-name">{{business.legalName}}</label>
```

Comment:
I think this needs to be `businessName` as `legalName` is not the right value for firms.  The existing `businessName` value might not work properly as is right now though.